app/uk/gov/hmrc/leakdetection/services/AlertingService.scala

```diff
@@ -143,8 +143,8 @@ class AlertingService @Inject()(configuration: Configuration,
   private def sendSlackMessage(slackNotificationAndErrorMessage: SlackNotificationAndErrorMessage)(
     implicit hc: HeaderCarrier): Future[Unit] =
     slackConnector.sendMessage(slackNotificationAndErrorMessage.request).map {
-      case SlackNotificationResponse(errors) if errors.isEmpty => ()
-      case SlackNotificationResponse(errors) =>
+      case SlackNotificationResponse(success, _) if success.nonEmpty => ()
```

Comment:
would `case resp if resp.sucessfullySentTo.nonEmpty` rather than case decomposition be less brittle? (maybe a derived field `resp.isSent`?)

Reply:
if you like